inlong-manager/manager-web/sql/apache_inlong_manager.sql

```diff
@@ -158,6 +158,40 @@ CREATE TABLE IF NOT EXISTS `inlong_cluster_node`
 ) ENGINE = InnoDB
   DEFAULT CHARSET = utf8mb4 COMMENT ='Inlong cluster node table';
 
+-- ----------------------------
+-- Table structure for inlong_label
+-- ----------------------------
+CREATE TABLE IF NOT EXISTS `inlong_label`
```

Comment:
Label is a word with no clear meaning. Can you give it a clear name according to the characteristics?